velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -25,58 +25,75 @@ namespace {
 
 class NthValueTest : public WindowTestBase {
  protected:
+  NthValueTest() : frameClause_("range unbounded preceding") {}
+
+  NthValueTest(const std::string& frameClause) : frameClause_(frameClause) {}
+
+  void testWindowFunction(
+      const std::vector<RowVectorPtr>& input,
+      const std::string& function,
+      const std::vector<std::string>& overClauses) {
+    WindowTestBase::testWindowFunction(
+        input, function, overClauses, frameClause_);
+  }
+
   void testPrimitiveType(const TypePtr& type) {
     vector_size_t size = 25;
     auto vectors = makeRowVector({
         makeFlatVector<int32_t>(size, [](auto row) { return row % 5; }),
         makeFlatVector<int32_t>(size, [](auto row) { return row; }),
         makeFlatVector<int64_t>(size, [](auto row) { return row % 3 + 1; }),
-        typeValues(type, size),
+        makeFlatFuzzVector(type, size),
```

Comment:
Why do we need this change ?

Reply:
Since ```makeFlatFuzzVector``` is identical to the ```typeValues``` function and ```makeFlatFuzzVector``` is used in NthValueTest and SimpleAggregatesTest, function ```typeValues``` is removed.

Reply:
Reverted, thanks for pointing it out. It was added earlier for testing with the five column over clauses.